www/manager/templates/html/Feed/Media/Form.tpl.php

```diff
@@ -36,7 +35,7 @@
         <h4>This Video is Missing Captions!</h4>
         <div class="mh-caption-band">
             <p>
-                MediaHub uses <a href="http://amara.org/en/">Amara</a> for the captioning of video. For accessibility reasons, captions are required for <strong>ALL</strong> videos.
+                For accessibility reasons, captions are required for <strong>ALL</strong> videos.
```

Comment:
Avoid ALL CAPS when the element is already wrapped in a semantically appropriate element. You can use CSS if you _really_ need it to be in all caps, internet shouter.